core/modules/layout/layout.admin.inc

```diff
@@ -94,7 +94,34 @@ function layout_list_page() {
     array('data' => t('Operations'), 'class' => array('layout-operations')),
   );
 
+  $help_items = array();
+  $help_items[] = t('To determine which layout is to be used for a page, the path is compared to those shown here.');
+  $help_items[] = t('Paths used by the layout system are <em>system paths</em> (also called <em>normal paths</em>), not !link.', array('!link' => l(t('URL aliases'), 'admin/config/urls/path')));
+  $help_items[] = t('They are listed alphabetically below. All layouts that might apply to each path are listed below the path.');
```

Comment:
"They"? ...which ones? ...the "system paths" or the "URL aliases"? ...or both? Let's please be explicit/specific here, to avoid confusion for people that are new to all of this.

Reply:
I agree, `They` here should be `Paths`.  It would also be clearer if we put the disambiguation after the explanation so move `Paths used by the layout system are...` below `They are listed alphabetically...`.